packages/db-base/src/lib/inMemFileDB.ts

```diff
@@ -39,12 +32,83 @@ const tools = require('./tools.js');
 // };
 //
 
+interface ConnectionOptions {
+    // relative path to the data dir
+    backup?: BackupOptions;
+    dataDir: string;
+}
+
+type ChangeFunction = (id: string, state: any) => void;
```

Comment:
What is `state`? By any chance an [`ioBroker.State`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/bc49c2f0d802b9b83f6d3e99e0cfe0a4f7733cf1/types/iobroker/index.d.ts#L34-L61)?

Reply:
don't hink so, because Objects DB also extends this class, "maybe" `ioBroker.Object` would be correct, but idk why it is/was named state

Reply:
would keep it as any for now